gitlens/gitlens-release-notes-current.md

```diff
@@ -16,6 +16,58 @@ Features marked with `PREVIEW` require a GitKraken Account, with access level ba
 
 ---
 
+
+<a id="v15-1"></a>
+
+## Version 15.1
+
+#### Tuesday, June 5th, 2024
+
+We're excited to announce the latest release of GitLens, bringing you enhanced AI integration, a better Launchpad experience, and bug fixes. This focus on expanding AI capabilities, streamlining user experience, and addressing issues reported by our community.
+
+<img src="/wp-content/uploads/gl-15-1-hero.png" class="img-responsive center img-bordered">
+
+### GitHub Copilot and AI Model Integration
+
+GitLens now supports integration with GitHub Copilot and other VS Code extension-provided AI models for its experimental AI features. You can now specify the AI model you want to use through the new `gitlens.ai.experimental.model` setting. When the model is set to `vscode`, specifying the `gitlens.ai.experimental.vscode.model` setting will target the VS Code extension-provided AI model you want to use.
```

Comment:
I'd suggest showing the picker or at least talking about it with the settings as secondary, since you should never have to touch the settings.